docs/topics/kubernetes-developers.md

```diff
@@ -123,3 +123,49 @@ AZURE_CLOUD_NODE_MANAGER_IMG=myrepo/my-cnm:v0.0.1 \
 CLUSTER_TEMPLATE=cluster-template-external-cloud-provider.yaml \
 make create-workload-cluster
 ```
+
+## Testing clusters built from Kubernetes source
+
+[A template is provided](../../templates/test/dev/cluster-template-custom-builds.yaml) that enables building clusters from custom built Kubernetes components, and assumes the following environment variables are declared, and their values injected using a tool like `envsubst`:
+
+- `$CLUSTER_NAME`
```

Comment:
thoughts on keeping only the variables that are specific to this template here to avoid duplicated information with other docs, and so we don't have to update it when something changes in the default template?

Reply:
Done